src/components/accession2/withdraw/WithdrawModal.tsx

```diff
@@ -310,7 +391,9 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
               errorText={fieldsErrors.withdrawnQuantity}
             />
             <Box paddingLeft={1}>
-              {accession.remainingQuantity?.units === 'Seeds' || record.purpose === 'Viability Testing' ? (
+              {accession.remainingQuantity?.units === 'Seeds' ||
+              record.purpose === 'Viability Testing' ||
+              isNurseryTransfer ? (
```

Comment:
If the accession is in grams with purpose nursery I'm still seeing the dropdown for units

Reply:
you will see that in old behavior - do you have nursery management enabled?
here's what it should look like 
<img width="527" alt="Screen Shot 2022-10-27 at 7 45 00 AM" src="https://user-images.githubusercontent.com/1865174/198320893-76fd0e61-481f-4670-a9d2-bc09efac1069.png">

Reply:
with nursery management disabled 
<img width="602" alt="Screen Shot 2022-10-27 at 7 46 48 AM" src="https://user-images.githubusercontent.com/1865174/198321529-4c0afc2b-ad34-42fd-a94b-acddcdad0198.png">

Reply:
I'm seeing correctly now. I had nursery management enabled but I don't know why it wasn't working. Now it is! Thanks

Reply:
I was trying to reproduce it and I couldn't. Maybe I was seeing an old version in Vercel